Sentinel mocks are available indefinitely (#1947)
hashicorp/terraform-website: content/source/docs/cloud/sentinel/mock.html.md

  produced via mock generation.
* The run has not progressed past the planning stage, or did not create a plan
  successfully.

> * The run had progressed past the planning stage prior to July 23, 2021. Prior to this date, JSON plans were only kept for 7 days.

Reviewer: Again, just want it to be active voice :-)

```suggestion
* The run progressed past the planning stage prior to July 23, 2021. Prior to this date, Terraform Cloud only kept JSON plans for 7 days.
```

Reviewer 2: I would rather see us use this:
```
The run progressed past the planning stage prior to July 23, 2021 and more than seven days had passed. (After that date, Terraform Cloud began permanently keeping the JSON plans that mock generation requires.)
```

Reviewer: I like your edit except - I think let's omit the parenthesis and say "the JSON plans required for mock generation". If we do that, I'm sold!

> -> **Note:** Only the plan needs to be successful for a run to be eligible for

Reviewer: Can we make it more active? Something like this: 

```suggestion
-> **Note:** Only a successful plan is required for mock generation. Sentinel can still generate the data if apply or policy checks fail.
```
